src/crs_linter/rules/deprecated.py

```diff
@@ -0,0 +1,34 @@
+from crs_linter.lint_problem import LintProblem
+from crs_linter.rule import Rule
+
+
+class Deprecated(Rule):
```

Comment:
Does this class have the same functionality like `ctl_audit_log` above? Why are both necessary?